ceph_deploy/install.py

```diff
@@ -584,6 +589,12 @@ def make(parser):
                 (defaults to ceph.com)'
     )
 
+    repo.add_argument(
+        '--nogpgcheck',
```

Comment:
@alfredodeza Yeah, it is useful when the "--local-mirror" or "--repo-url" is specified to leave the ceph-deploy do other repo configurations.

Thanks,
Wanlong